bundle/language_en.properties

```diff
@@ -335,6 +335,7 @@ angal.billbrowser.shortreportonlybaddebt.txt
 angal.billbrowser.thebilldeleted.msg                                                                   = The bill is deleted.
 angal.billbrowser.thebillisstillopen.msg                                                               = This bill is still open.
 angal.billbrowser.thismonth.txt                                                                        = This Month
+angal.billbrowser.to                                                                                   = To
```

Comment:
Why isn't the common version used instead of creating a new key; **angal.common.to.txt**